lib/resources/r4/financial_transaction.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: FinancialTransaction
+field_name_base_url: http://hl7.org/fhir/R4/basic.html
+fields:
+- name: financial-transaction-method
+  description: Describes the method of payment for the financial transaction.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-method
+  binding:
+    description: The payment method of the transaction.
+    terminology:
+    - display: v2 Payment Method Code
+      system: https://terminology.hl7.org/CodeSystem/v2-0570
+      info_link: https://www.hl7.org/fhir/v2/0570/index.html
+
+- name: financial-transaction-type
+  description: A value associated to the specific payment method usually represented as the last four digits of a credit card, the check number, the EFT number or Lockbox number.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-type
+  binding:
+    description: The Category of Payment Type Codes.Whether this is a payment or adjustment.
```

Comment:
nit: need a space after the period

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4adf16f12daf8a74d8b1caef1c28f22dd3d0b8fd#diff-5125c86cde87f8439cb9d1bb859194e9R19

Reply:
+1